website/content/api-docs/acl/auth-methods.mdx

```diff
@@ -433,3 +438,15 @@ listing and must be retrieved by the [auth method reading endpoint](#read-an-aut
   }
 ]
 ```
+
+## Methods to Specify Namespace <EnterpriseAlert inline />
```

Comment:
Many of these pages have endpoints that support several ways of specifying a namespace with the same order of precedence. I decided to combine these into one clearer section that is referenced (from `ns` query param and `Namespace` JSON request body fields) rather than the previous solution (which repeated a less clear description every time the ns/Namespace parameters were listed).

Let me know if there's a better way to handle this.